src/writeData/components/ClientCodeCopyPage/InstallPackageHelper.tsx

```diff
@@ -13,14 +13,13 @@ import {
   ComponentSize,
 } from '@influxdata/clockface'
 import {MarkdownRenderer} from 'src/shared/components/views/MarkdownRenderer'
-import {Renderer} from 'react-markdown'
 
 // Styles
 import 'src/writeData/components/ClientCodeCopyPage/InstallPackageHelper.scss'
 
 interface Props {
   text: string
-  codeRenderer: Renderer<HTMLPreElement>
+  codeRenderer: any
```

Comment:
only concern here is that we're losing type information going from `Renderer` to `any`. Is there any way to avoid this?

Reply:
Hrmm a valid concern for sure but I'm going to have to punt on it and defer to https://github.com/influxdata/ui/pull/3039 which is where this code is from, and matches the impl on `master`: https://github.com/influxdata/ui/blob/9edf248bb259dd4dfa49b9ebc7f9a65b84c18b02/src/writeData/components/ClientCodeCopyPage/InstallPackageHelper.tsx#L17-L20

I'd guess that there were some changes in the new version of `react-markdown` that made `Rendered` no longer compatible, but I can't say for sure. For now I'll keep this code consistent with `master`, and if it gets changed there in the future we'll pick it up when we create new OSS release branches off of `master`.